Paraswap-Agents/Role-Changes/README.md

```diff
@@ -0,0 +1,45 @@
+# Role Change Detection Agent
+
+## Description
+
+This agent detects role changes in `AccessControl` contract.
```

Comment:
The agent actually detects role changes in the `AugustusSwapper` contract. The contract `AccessControl` is defined, but it's actually inherited by `AugustusSwapper`.
I'd recommend changing all references to `AccessControl` to `AugustusSwapper` instead.